### PR TITLE
Fix apk sf mf line length

### PR DESCRIPTION
**Pull request by g-k**
refs prior email thread with rbarker

r? @jvehent

### CODE REVIEW

File: signer/apk/jar.go [hunk 1]
```diff
@@ -9,8 +9,56 @@ import (
...
+		err = errors.Errorf("apk: invalid UTF8 in filename %s", filename)
+		return
+	}
+
+	const (
+		maxLineByteLen = 72
+		maxContinuedByteLen = 70 // -1 for leading space and -1 for trailing \n newline
+	)
+	var (
+		maxFirstLineByteLen = maxLineByteLen - (len([]byte("Name: ")) + 1)  // + 1 for a \n newline
```
Comment (jvehent) [inline signer/apk/jar.go]: nit: recalculating this at every execution seems like a waste

File: signer/apk/jar.go [hunk 2]
```diff
@@ -9,8 +9,56 @@ import (
...
+		fileBytes = len(filename)
```
Comment (jvehent) [inline signer/apk/jar.go]: nit: is `fileBytes` the right name for this, or should it be `filenameLen`?

**Comment by g-k** [comment]
I do wonder if this applies to XPIs too or if XPIs aren't that strict about following the JAR format.

**Comment by jvehent** [comment]
This is definitely something we should test. I'm inclined to think Firefox doesn't care, since we have hundreds of thousands of addons and never had a report of breakage, but data is better than intuition.

**Comment by g-k** [comment]
Yeah, I'll ask on #amo and check the python signer source.

**Pull request merged**